### PR TITLE
Mention aiohttp_jinja2 in the docs

### PR DESCRIPTION
**Pull request by asvetlov**
Add reference to `aiohttp_jinja2` library in `aiohttp.web` usage instructions.

### CODE REVIEW

File: docs/web.rst [hunk 1]
```diff
@@ -174,6 +174,34 @@ The next example shows custom processing based on *HTTP Accept* header:
    chooser.reg_acceptor('application/xml', handle_xml)
 
 
+Template rendering
+------------------
+
+:mod:`aiohttp.web` has no support for template rendering out-of-the-box.
+
+But there is third-party library :mod:`aiohttp_jinja2` which is
+supported by *aiohttp* auhtors.
```
Comment (ludovic-gasc) [inline docs/web.rst]: You've a typo in `auhtors` word.

Comment (asvetlov) [inline docs/web.rst]: Fixed, thanks.

File: docs/glossary.rst [hunk 2]
```diff
@@ -19,3 +19,7 @@
 
       Any object that can be called. Use :func:`callable` to check
       that.
+
+   web-handler
+
+       An enpoint that returns http response.
```
Comment (kxepal) [inline docs/glossary.rst]: endpoint?

Comment (asvetlov) [inline docs/glossary.rst]: Fixed, thanks!

**Comment by fafhrd91** [comment]
lgtm

**Pull request merged**